### PR TITLE
Proposed changes to application/manage

### PR DESCRIPTION
**Pull request by rjshanahan**
Move all declared financials to application/manage endpoint

### CODE REVIEW

File: XaiValidatePublicApiDefinition.yml [hunk 1]
```diff
@@ -3116,6 +2876,11 @@ components:
                     format: double
                     description: Represents the net surplus ratio including addbacks at application level.
                     example: 1.6611
+                  MaxBorrowingAmount:
```
Comment (jync) [inline XaiValidatePublicApiDefinition.yml:2881]: @rjshanahan could we clarify what this is about?

File: XaiValidatePublicApiDefinition.yml [hunk 2]
```diff
@@ -4172,12 +3944,20 @@ components:
                         HasHecsDebt:
                           type: boolean
                           description: Determines whether the applicant has a HECS debt to pay.
+                        MaritalStatus:
```
Comment (jync) [inline XaiValidatePublicApiDefinition.yml:3947]: @rjshanahan could you explain why this is necessary?

File: XaiValidatePublicApiDefinition.yml [hunk 3]
```diff
@@ -4315,6 +4125,19 @@ components:
...
+                                  HomeLoanTerm:
+                                    type: number
+                                    description: Declared home loan term.
+                                  HomeLoanRepaymentType:
+                                    type: string
+                                    description: Declared home loan repayment type.
+                                    enum:
+                                    - PI
+                                    - IO
+                                    - LOC
```
Comment (jync) [inline XaiValidatePublicApiDefinition.yml:4140]: @rjshanahan could we clarify what LOC is?

File: XaiValidatePublicApiDefinition.yml (hunk 1 above)
Comment (rjshanahan) [reply XaiValidatePublicApiDefinition.yml:2881]: @jync this is the maximum borrowing amount the team requested. We are now exposing this in the serviceability response.

File: XaiValidatePublicApiDefinition.yml (hunk 2 above)
Comment (rjshanahan) [reply XaiValidatePublicApiDefinition.yml:3947]: @jync `MaritalStatus` is used to drive the HEM lookup value (along with income and dependants)

File: XaiValidatePublicApiDefinition.yml (hunk 3 above)
Comment (rjshanahan) [reply XaiValidatePublicApiDefinition.yml:4140]: @jync `LOC` is for Line of Credit home loans - rare but they're still out there, and required different repayment calc treatment

**Pull request merged**